app/workers/commit_monitor_handlers/batch/github_pr_commenter/migration_date_checker.rb

```diff
@@ -0,0 +1,44 @@
+require 'time'
+
+module CommitMonitorHandlers::Batch
+  class GithubPrCommenter::MigrationDateChecker
```

Comment:
I didn't want 5 levels of indent...it's all going to change anyway when I drop the "type" part